livespark-showcase/livespark-webapp/pom.xml

```diff
@@ -43,6 +43,12 @@
   </properties>
 
   <dependencies>
+    <!-- drools-compiler has dependency on org.eclipse.jdt.core.compiler:ecj:jar:3.5.1:compile, see http://code.google.com/p/google-web-toolkit/issues/detail?id=4479 -->
```

Comment:
@psiroky @manstis can't comment on ecj for drools but this is still needed so the right version of jdt is on the classpath during gwt compilation. Would you prefer I remove the comment on all PRs?